python/pyspark/sql/conf.py

```diff
@@ -64,6 +64,96 @@ def _checkType(self, obj, identifier):
                             (identifier, obj, type(obj).__name__))
 
 
+class ConfigEntry(object):
```

Comment:
@gatorsmile I moved all core configs using in pyspark into conf.py here. Please have a look when you have time.
https://github.com/apache/spark/pull/21370#discussion_r194276735

Reply:
Thank you for fixing this! Let us do it in a separate PR.

Reply:
Yep, done in #21648